src/amber/cli/templates/scaffold/controller/granite/src/controllers/{{name}}_controller.cr.ecr

```diff
@@ -63,4 +63,14 @@ class <%= @name.capitalize %>Controller < ApplicationController
     end
     redirect_to "/<%= @name %>s"
   end
+
+  def <%= @name %>_params
+	params.validation do
+	  <% @fields.each do |field| %>
+	  <% unless field.cr_type() != "time" %>
+	  required(:<%= field.name %>) { |p| !p.as(<%= field.cr_type.capitalize %>) }
```

Comment:
I'm not following this logic.  Why would all the fields be required except times?

Reply:
This are the incoming URL/Form/Body params  that gets parsed. The time types are in reference to 'created_at' and 'updated_at' maybe it should be checking for those specific fields. 🤔

Reply:
yeah, I think you should iterate the `@visible_fields` since it takes care of which fields would be passed in the params.  There may be other fields that are `time` fields that are not created_at or updated_at.

Reply:
Unfortunately `@visible_fields` is an array of only names (String) and I need the type as well.